packages/zoe/src/contractSupport/priceAuthorityInitial.js

```diff
@@ -110,7 +110,7 @@ export const makeInitialTransform = (
       : quoteP;
   };
 
-  return Far('PriceAuthority', {
+  return Far('ScaledPriceAuthority', {
```

Comment:
why not `PriceAuthorityInitial` like on the tin?

Reply:
`priceAuthorityInitial` is a special case for when an `initialPrice` is provided in the configuration. You should only see that in testing scenarios. It shouldn't show up in production. The production `Far` object comes from `packages/zoe/src/contractSupport/priceAuthorityTransform.js`. So `ScaledPriceAuthorityWithInitial` would make more sense here.

Reply:
Okay, then I'll rotate the terms so the one we see in production will be `ScaledPriceAuthority`.